src/job_cache/daemon_cache.cpp

```diff
@@ -72,6 +72,29 @@ static void initialize_logging() {
   }
   wcl::log::subscribe(std::make_unique<JsonSubscriber>(std::move(*log_file_res)));
 
+  const char *bulk_dir = getenv("WAKE_BULK_LOGGING_DIR");
+  if (bulk_dir) {
+    std::string pid = std::to_string(getpid());
+    char buf[512];
+    if (gethostname(buf, sizeof(buf)) != 0) {
+      std::cerr << "urgent warning: Could not init logging: gethostname(): " << strerror(errno)
```

Comment:
lol "urgent warning" instead of "error" is a funny concept but I get why its called that. The program doesn't exit.